R/cal-validate.R

```diff
@@ -0,0 +1,239 @@
+#' Obtain and validate performance metrics from calibration
+#' @details These functions take an re-sampled object, created via `rsample`,
+#' and for each re-sample, it calculates the calibration on the training set, and
+#' then applies the calibration on the assessment set. By default the average of
+#' Brier scores is returned. It compares the average of the metrics before, and
+#' after the calibration.
+#' @param metrics A set of metrics passed created via `yardstick::metric_set()`
+#' @param summarize Indicates to pass tibble with the metrics averaged, or
+#' if to return the same sampled object but with new columns containing the
+#' calibration y validation list columns.
+#' @examples
+#'
+#' library(magrittr)
+#'
+#' segment_logistic %>%
+#' rsample::vfold_cv() %>%
+#'   cal_validate_isotonic(Class)
+#'
+#' segment_logistic %>%
+#'   rsample::mc_cv() %>%
+#'   cal_validate_beta(Class)
+#'
+#' segment_logistic %>%
+#'   rsample::bootstraps() %>%
+#'   cal_validate_logistic(Class)
+#'
+#' segment_logistic %>%
+#'   rsample::bootstraps() %>%
+#'  cal_validate_isotonic(Class, summarize = FALSE)
+#'
+#' @inheritParams cal_estimate_logistic
```

Comment:
This gives us 

> A data.frame object, or `tune_results` object, that contains predictions and probability columns.

There are not methods for `tune_results` (and that's ok with me)

Reply:
Not sure what the action is here